packages/@lwc/compiler/src/rollup-plugins/minify.ts

```diff
@@ -5,42 +5,29 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
  */
 import { Plugin } from 'rollup';
-
-import { NormalizedOutputConfig } from '../options';
+import { minify } from 'terser';
 
 /**
  * Rollup plugin applying minification to the generated bundle.
  */
-export default function ({ sourcemap }: NormalizedOutputConfig): Plugin {
-    // Inlining the `terser` module require to only pay the parsing and evaluation cost for needed
-    // modules
-    const { minify } = require('terser');
-
+export default function ({ sourcemap }: { sourcemap: boolean }): Plugin {
     return {
         name: 'lwc-minify',
 
-        renderChunk(src: string) {
-            const { code, map, error } = minify(src, {
+        async renderChunk(src: string): Promise<{ code: string; map?: any }> {
```

Comment:
When I tried this locally, it seemed like we have different version of the source-map library at the root and in the @lwc/compiler subpackage.
```suggestion
        async renderChunk(src: string): Promise<{ code: string; map?:  RawSourceMap | string }> {
```

Reply:
The source map types are not compatible cross libraries. Since the source maps are not directly touched by the compiler, I don't think it's a big issue that it is typed as `any`.